src/type-mapper.cc

```diff
@@ -24,7 +24,13 @@ TypeMapper::infer_type(const Local<Value>& value)
         return CASS_VALUE_TYPE_BOOLEAN;
     }
     else if (value -> IsObject()) {
-        return CASS_VALUE_TYPE_MAP;
+        // If the value has a low/high key it's a bigint, otherwise it's a map
```

Comment:
For that matter I think you should be able to just add another entry to the test case table in `hints.spec.js` instead of needing a dedicated test for the basic support for bigint.

Testing that `writetime` works does seem worthy of its own test.